lib/src/main/java/edu/stanford/futuredata/macrobase/analysis/classify/ArithmeticClassifier.java

```diff
@@ -0,0 +1,162 @@
+package edu.stanford.futuredata.macrobase.analysis.classify;
+
+import edu.stanford.futuredata.macrobase.datamodel.DataFrame;
+
+import java.lang.Double;
+import org.apache.commons.math3.stat.descriptive.rank.Percentile;
+import org.apache.commons.math3.distribution.NormalDistribution;
+
+/**
+ * Classify rows by high / low values based on the group mean and standard deviation.
+ * Returns a new dataframe with a column representation of the estimated number of outliers
+ * for each group, which can be non-integer.
+ */
+public class ArithmeticClassifier extends CubeClassifier implements ThresholdClassifier {
+    // Parameters
+    private String meanColumnName = "mean";
+    private String stdColumnName = "std";
+    private double percentile = 1.0;
+    private boolean includeHigh = true;
+    private boolean includeLow = true;
+
+    // Calculated values
+    private double lowCutoff;
+    private double highCutoff;
+    private DataFrame output;
+
+    public ArithmeticClassifier(String countColumnName, String meanColumnName,
+                                String stdColumnName) {
+        super(countColumnName);
+        this.meanColumnName = meanColumnName;
+        this.stdColumnName = stdColumnName;
+    }
+
+    @Override
+    public void process(DataFrame input) {
+        double[] means = input.getDoubleColumnByName(meanColumnName);
+        double[] counts = input.getDoubleColumnByName(countColumnName);
+        double[] stds = input.getDoubleColumnByName(stdColumnName);
+        int len = means.length;
+        int numRawMetrics = 0;
```

Comment:
hmm but then it's awkward when I use these variables to set the length of arrays and I have to cast them back to ints